### PR TITLE
mail/postfix: add newaliases to setup.sh

### PR DESCRIPTION
**Pull request by mimugmail**
to prevent error logging

### CODE REVIEW

File: mail/postfix/src/opnsense/scripts/OPNsense/Postfix/setup.sh [hunk 1]
```diff
@@ -31,4 +31,15 @@ postmap /usr/local/etc/postfix/transport
...
 postmap /usr/local/etc/postfix/sender_access
 
+# Check for aliases
+if [ -f /usr/local/etc/postfix/aliases ]; then
+       echo "Updateing aliases"
+       /usr/local/bin/newaliases
+else
+       touch /usr/local/etc/postfix/aliases
+       /usr/local/bin/newaliases
+       echo "Added aliases file"
```
[the remark this answers is not in the document]
Comment (mimugmail) [reply mail/postfix/src/opnsense/scripts/OPNsense/Postfix/setup.sh]: fixed

**Comment by mimugmail** [comment]
added syslog tweak for testing ..

**Comment by mimugmail** [comment]
I really wrote updateing :D

**Comment by fichtner** [comment]
Thanks!

**Pull request merged**